src/platform/forms/components/review/FormSaveErrorMessage.jsx

```diff
@@ -76,8 +76,9 @@ function FormSaveErrorMessage(props) {
       >
         <p>
           We’re working to fix the problem. Please make sure you’re connected to
-          the Internet, and then try saving your {appType} again. {saveLink}.
+          the Internet, and then try saving your {appType} again.
         </p>
+        {saveLink}.
```

Comment:
I know this is out of scope, but `saveLink` includes a `div` and console errors showing no `div` inside a `p` got annoying.